test/teams_test.rb

```diff
@@ -14,7 +14,7 @@ def setup
     franchiseId: "23",
     teamName: "Atlanta United",
     abbreviation: "ATL",
-    Stadium: "Mercedes-Benz Stadium",
+    stadium: "Mercedes-Benz Stadium",
```

Comment:
Atom was seeing "Stadium" as a class due to first letter capitalized. Renamed - need to verify that column conversion to symbol still maps correctly.